staticd/static_vrf.c

```diff
@@ -100,6 +100,9 @@ static int static_vrf_delete(struct vrf *vrf)
 	for (afi = AFI_IP; afi <= AFI_IP6; afi++) {
 		for (safi = SAFI_UNICAST; safi <= SAFI_MULTICAST; safi++) {
 			table = svrf->stable[afi][safi];
+			if (!table)
```

Comment:
This may fix a crash for you, but it's not a correct fix.

`svrf->stable[afi][safi]` is always initialized in the exactly same loop – look into `static_vrf_alloc`.
If you see that `table` is NULL for you then we have some memory corruption that we'll never fine if we merge this.

Reply:
I actually think this is just another form of the crash that is fixed by https://github.com/FRRouting/frr/pull/9943
We're calling `static_vrf_delete` for already deleted VRF so you see that the pointer is NULL.

Reply:
yeah run topotests and look in /var/tmp/frr.

In any event we are calling the shutdown twice that is why it is happening.  I don't think it's possible to easily unwind hence this test

Reply:
#9943 fixes the double deleting of the VRF